torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -113,6 +113,9 @@ def placeholder(self, target, args, kwargs):
             proxy.node.meta = self._input_node_meta[proxy.node.name]
         return proxy
 
+    def get_new_graph_node_meta(self):
+        return {node.name: node.meta for node in self.new_graph.nodes}
```

Comment:
Where does the `self.new_graph` come from?

Reply:
I think the mechanism of `torch.fx.Transformer` is one by one transom origin graph's op to `new_graph`
https://pytorch.org/docs/stable/fx.html#torch.fx.Transformer
Here's its implement code
https://pytorch.org/docs/stable/_modules/torch/fx/interpreter.html#Transformer

The problem I encounter here is:
 original graph have 8 getitem
```
getitem0 = max_pool_0[0]
getitem1 = max_pool_0[1]
getitem2 = max_pool_1[0]
getitem3 = max_pool_1[1]
...
```
And the input of `adaptive_avg_pool` is
```
adaptive_avg_pool_0.args[0] = getitem0
adaptive_avg_pool_1.args[0] = getitem2
```
And `self._input_node_meta`  record these original graph's information

But the `new_graph`, due to `getitem1` & `getitem3` has no user, so maybe `remove_unused_op` method is applied, so the `new_graph` is
```
getitem0 = max_pool_0[0]
getitem1 = max_pool_1[0]
adaptive_avg_pool_0.args[0] = getitem0
adaptive_avg_pool_1.args[0] = getitem1
```
So in the lowering of `adaptive_avg_pool`, these code is wrong
```
# in adaptive_avg_pool_1
input_name = node.args[0].name # getitem1, it actually from max_pool_1[0]
input_shape = self._input_node_meta[input_name] # access origin graph and get max_pool_0[1], which is wrong!
```

So cannot use `self._input_node_meta`, use `get_new_graph_node_meta` instead

And because `new_graph` is generting at this moment, so it cannot set at `__init__`, instead use `get_new_graph_node_meta` to get the real time updates

Reply:
Updated:
After discuss with @jerrysky3 , turns out just use `node.args[0].node.meta` can satisfy the requiremt, so no need the `get_new_graph_node_meta` function